index.js

```diff
@@ -33,9 +33,11 @@ const TALKING_POINT_TARGET_CHANNEL_ID = process.env.TALKING_POINT_CHANNEL_ID;
 const GENERAL_CHANNEL_ID = process.env.GENERAL_CHANNEL_ID;
 const USERNAME_REGEX = /<@(\d+)>/g;
 const TODO_REGEX = /TODO/;
+const TALKING_POINT_REGEX = /(Talking point|TP)/;
```

Comment:
Keep in mind this is case-sensitive. If it should be case insensitive, then you can add an `i` past the `/` in the regex to make it case insensitive

Reply:
Ack